scripts/sign.py

```diff
@@ -144,7 +144,7 @@ def main():
     h = SHA256.new()
 
     digest_len = h.digest_size
-    sig_len = ceil_div(key.size() + 1, 8)
+    sig_len = ceil_div(size(key.n) + 1, 8)
```

Comment:
If this now is considered unsafe, perhaps we should double check this prediction matches the actual result when writing the image below in `write_image_with_signature()`

Reply:
Good idea. Will do in the next version.